app/views/agents/_form.html.haml

```diff
@@ -56,27 +54,29 @@
     %th
       = t("agents.form.identifiers")
     %td.top
-      %div.agents-identifiers
+      %div.agents-identifiers{'data-form-options-display-target':"option1",  class: affiliation?(agent) && 'd-none'}
         = render NestedFormInputsComponent.new do |c|
           - c.template do
             %div.d-flex{id: 'NEW_RECORD'}
               %div.w-100
                 = hidden_field_tag agent_identifier_name('NEW_RECORD' , :creator, name_prefix), session[:user].id
-                = text_field_tag agent_identifier_name('NEW_RECORD' , :notation, name_prefix), '', class: "form-control"
-              %div{style:'width: 15%'}
-                - values = %w[ORCID ROR ISNI GRID]
-                = render SelectInputComponent.new(id: "agent_identifiers_schemaAgency_NEW_RECORD", name: agent_identifier_name('NEW_RECORD', :schemaAgency, name_prefix), values: values, selected: 'ORCID')
+                = inline_svg_tag 'orcid.svg', class: 'agent-input-icon'
```

Comment:
create a re-usable helper, used here and in the account edit page.

Reply:
We use it only here and in the signup page, and the css classes are different I see that we'll endup just complexing the code